sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -64,15 +62,18 @@
 )
 from ._list_blobs_helper import BlobPrefix
 
+if TYPE_CHECKING:
+    from azure.core.credentials import AzureNamedKeyCredential, AzureSasCredential, TokenCredential
+
 __version__ = VERSION
 
 
 def upload_blob_to_url(
-        blob_url,  # type: str
-        data,  # type: Union[Iterable[AnyStr], IO[AnyStr]]
-        credential=None,  # type: Optional[Union[str, Dict[str, str], AzureNamedKeyCredential, AzureSasCredential, "TokenCredential"]] # pylint: disable=line-too-long
-        **kwargs):
-    # type: (...) -> Dict[str, Any]
+    blob_url: str,
+    data: Union[Iterable[AnyStr], IO[AnyStr]],
+    credential: Optional[Union[str, Dict[str, str], "AzureNamedKeyCredential", "AzureSasCredential", "TokenCredential"]] = None,  # pylint: disable=line-too-long
```

Comment:
`azure\storage\blob\aio\__init__.py:75: error: Returning Any from function declared to return "Dict[str, Any]"  [no-any-return]`